test/test-ce.html

```diff
@@ -8,7 +8,11 @@
     <script src="/src/fragment-ce.js" async></script>
   </head>
   <body>
+    <x-fragment config>
+      <config header cache-control="must-revalidate"></config>
+    </x-fragment>
```

Comment:
So, @Anomeon what you proposed is to write
```html
<meta x-fragment header cache-control="must-revalidate">
```
instead of 
```html
<x-fragment config>
  <config header cache-control="must-revalidate"></config>
</x-fragment>
```

Sounds fair...